e2e/dragonfly_controller_test.go

```diff
@@ -305,7 +305,7 @@ var _ = Describe("Dragonfly Lifecycle tests", Ordered, FlakeAttempts(3), func()
 			}, &df)
 			Expect(err).To(BeNil())
 
-			df.Spec.Image = fmt.Sprintf("%s:%s", resources.DragonflyImage, "v1.16.0")
```

Comment:
if you want to update df version you also need to do that at other places 🤔

Reply:
Just wanted to test the latest version before updating the default image, will drop this part and create a new PR for version upgrade.